core/src/main/java/com/datastrato/gravitino/GravitinoEnv.java

```diff
@@ -57,6 +58,11 @@ public static GravitinoEnv getInstance() {
     return InstanceHolder.INSTANCE;
   }
 
+  @VisibleForTesting
```

Comment:
Can you please add some comments on this method mention that this is only for test.

Reply:
I have updated the Java doc regarding it.